lib/config.js

```diff
@@ -80,10 +80,15 @@ const conf = convict({
     }
   },
   expiration: {
-    code: {
+    accessTokenS: {
```

Comment:
I like millisecond timestamps too, but I think it will be against the grain when interfacing with existing OAuth, JWT and other related specs, which all seem to use timestamps in seconds.

For example, this was one of the trickest parts of trying to bring the legacy BrowserID assertion format into line with the final JWT spec.  It caused a pretty funny (and thankfully low-impact) security bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1096106

In the interests of consistency, I suggest we grit our teeth and use integer second timestamps throughout the public API.